cypress/e2e/wallet-dropdown.test.ts

```diff
@@ -77,4 +77,12 @@ describe('Wallet Dropdown', () => {
     cy.get(getTestSelector('theme-auto')).click()
     cy.get(getTestSelector('wallet-header')).should('have.css', 'color', 'rgb(119, 128, 160)')
   })
+
+  it('should use a bottom sheet and dismiss when on a mobile screen size', () => {
+    visit(true)
+    cy.viewport('iphone-6')
```

Comment:
why not use an alias like this in landing.test.ts? Then you wouldn't need *_DIMENSIONS.